etl/storage/storage_metrics.go

```diff
@@ -38,6 +38,11 @@ var (
 		Help:       "The duration of the GetRecord function execution",
 		Objectives: quntitile,
 	})
+	GetStateDuration = prometheus.NewSummary(prometheus.SummaryOpts{
+		Name:       "gbe_storage_stats_GetStateDuration",
```

Comment:
was the metric added to the dashboard?

Reply:
It will be done in this task https://insolar.atlassian.net/browse/PENV-811